.circleci/config.yml

```diff
@@ -0,0 +1,77 @@
+version: 2.0
```

Comment:
Do you even run this on circle CI?

Reply:
That's the plan. I want to setup CI jobs so that we can run some tests and ensure we won't break things while we apply some refactoring.